remotes/transport/http.go

```diff
@@ -30,6 +31,11 @@ var (
 		Code: errors.EInvalid,
 		Msg:  "remote-connection ID is invalid",
 	}
+
+	errForeignKey = &errors.Error{
+		Code: errors.EInternal,
+		Msg:  "remote-connection cannot be deleted as a replication references it",
```

Comment:
This error response could probably be worded better, would appreciate feedback

Reply:
Sounds fine to me :+1: